frontend/my-app/src/components/CreateMeeting.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+
+export default function CreateMeeting() {
+    const [meetingInfo, setInfo] = useState({
+        code: "",
+        duration: "",
+        type: ""
+    })
+    const [isPending, setPending] = useState(false);
+    function submit(e) {
+        e.preventDefault();
+        setPending(true);
+        const data = {
+            code: meetingInfo.code,
+            duration: meetingInfo.duration,
+            type: meetingInfo.type
+        }
+        console.log(data);
+        fetch('http://localhost:5000/meeting', {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify(data)
+        })
+            .then(response => response.text())
+            .then(text => console.log(text))
+            .then(data => {
+                console.log('Success:', data);
+            })
+            .catch((error) => {
+                console.log('Error:', error);
+            })
+        setPending(false);
```

Comment:
I assume you're trying to prevent people from spam clicking the button, disabling it until the request has been completed. 

If this is the case, the isPending functionality does not really work. 

This is because you're calling fetch asynchronously and the application does not wait until fetch resolved to setPending to be false.

To illustrate, I wrote some debug code and have the request fail and logged when things happened. As you can see, "DONE" is printed before "Error: ... "  is

Does this all make sense?

![image](https://user-images.githubusercontent.com/39445369/158099708-86914901-3637-40e4-80f7-2cce1dc5a1ff.png)

![image](https://user-images.githubusercontent.com/39445369/158099748-2198117d-2520-461f-a8c1-b890092dc4ee.png)